.changeset/gorgeous-dryers-wash.md

```diff
@@ -0,0 +1,7 @@
+---
+'@shopify/polaris': minor
+'@shopify/polaris-tokens': minor
+---
+
+- Add border width prop to `Box`
+- Export color token subset alias types from tokens package and remove from `Box`
```

Comment:
I don't know if we're still trying to do past tense with these but I think it makes more sense in the changelog 🤷 

```suggestion
- Added border width prop to `Box`
- Exported color token subset alias types from tokens package and remove from `Box`
```